tokens-studio/figma-plugin
# fix: non prefix themes styling issue

Follow Up PR to fix styling issue where styles are not being applied when themes are being exported as non-prefixed styles



# Description

Fixes the issue where styles are not being applied when themes are being exported as non-prefixed styles

## Type of change


# How to test this

- Create some themes with enabled sets in the plugin
- Export themes as styles
- Make sure to uncheck the 'Prefix styles with active theme name' option
- Choose the themes to export as styles and export to Figma
- Apply token from a particular theme which has been exported
- The corresponding style should now be applied

# Screenshots or video (if necessary):
